x/stablestake/keeper/params.go

```diff
@@ -27,23 +26,3 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	b := k.cdc.MustMarshal(&params)
 	store.Set(types.ParamKeyPrefix, b)
 }
-
-func (k Keeper) GetDepositDenom(ctx sdk.Context) string {
```

Comment:
We are not using params.DepositDenom and params.RedemptionRate anymore right?
Lets rename it to LegacyDepositDenom and LegacyRedemptionRate so that it becomes its not being used